packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { AuthInfo, Connection } from '@salesforce/core';
+import * as vscode from 'vscode';
+import {  WorkspaceContextUtil } from '../../../src';
+
+describe('WorkspaceContext', () => {
+  const testUser = 'test@test.com';
+  const testAlias = 'TestOrg';
+  const testUser2 = 'test2@test.com';
+
+  let getUsernameStub: jest.SpyInstance;
+  let getUsernameOrAliasStub: jest.SpyInstance;
+  let workspaceContextUtil: any; // TODO find a better way
+  let authUtil: any;
+
+  const mockWatcher = {
+    onDidChange: jest.fn(),
+    onDidCreate: jest.fn(),
+    onDidDelete: jest.fn()
+  };
+  let mockFileSystemWatcher: jest.SpyInstance;
+
+  beforeEach(async () => {
+    mockFileSystemWatcher = (vscode.workspace
+      .createFileSystemWatcher as any).mockReturnValue(mockWatcher);
+
+    const context = {
+      subscriptions: []
+    };
+
+    workspaceContextUtil = WorkspaceContextUtil.getInstance(true);
+
+    authUtil = workspaceContextUtil.getAuthUtil();
+    getUsernameOrAliasStub = jest
+      .spyOn(authUtil, 'getDefaultUsernameOrAlias')
+      .mockReturnValue(testAlias);
+    getUsernameStub = jest
+      .spyOn(authUtil, 'getUsername').mockReturnValue(testUser);
+
+    await workspaceContextUtil.initialize(context);
+  });
+
+  afterEach(() => jest.clearAllMocks());
```

Comment:
this shouldn't be necessary as we do this automatically via config [here](https://github.com/forcedotcom/salesforcedx-vscode/blob/b257dbe6cf0bacec41095640fdd16ec3d0c7312c/config/jest.base.config.js#L12)